vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/internal/CwtStatusListTokenPayload.kt

```diff
@@ -0,0 +1,52 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList.internal
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.StatusList
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.StatusListTokenPayload
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.cwt.claims.CwtStatusListPayloadClaimSpecification
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.cwt.claims.CwtTimeToLivePayloadClaimSpecification
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.CwtTimeToLive
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.TimeToLive
+import at.asitplus.wallet.lib.data.rfc7519.primitives.ExpirationTime
+import at.asitplus.wallet.lib.data.rfc7519.primitives.IssuedAt
+import at.asitplus.wallet.lib.data.rfc7519.primitives.StringOrUri
+import at.asitplus.wallet.lib.data.rfc8392.cwt.claims.CwtExpirationTimePayloadClaimSpecification
+import at.asitplus.wallet.lib.data.rfc8392.cwt.claims.CwtIssuedAtPayloadClaimSpecification
+import at.asitplus.wallet.lib.data.rfc8392.cwt.claims.CwtSubjectPayloadClaimSpecification
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.cbor.CborLabel
+
+@Serializable
+internal data class CwtStatusListTokenPayload(
+    @CborLabel(CwtSubjectPayloadClaimSpecification.KEY)
+    @SerialName(CwtSubjectPayloadClaimSpecification.NAME) val subject: StringOrUri,
```

Comment:
Please reformat to keep in line with existing classes:

```kotlin

    @CborLabel(CwtSubjectPayloadClaimSpecification.KEY)
    @SerialName(CwtSubjectPayloadClaimSpecification.NAME)
    val subject: StringOrUri,
```

Reply:
Is extracting the key and name to a separate class (which involves another click/action when reading this class) really worth it? Also, the documentation about the field itself is also not here (what the `subject` means)

Reply:
Done

> Is extracting the key and name to a separate class (which involves another click/action when reading this class) really worth it

That seems like a highly subjective question with no absolute answer...
I tried separating the rfc details from the rest of the implementation as I like a high level of separation of concerns.
Manually writing the label is certainly possible, but having it like this seems to be possible as well. Since this class is purely internal, and shouldn't really be touched even within the library outside of the `tokenStatusRfc`-directory (use StatusListTokenPayload instead), I think this is fine.